Kickstarter-iOS/Views/Storyboards/Login.storyboard

```diff
@@ -317,7 +157,10 @@
                                                                 </constraints>
                                                             </view>
                                                             <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="JvF-DM-f0p">
-                                                                <rect key="frame" x="0.0" y="524" width="360" height="33"/>
+                                                                <rect key="frame" x="0.0" y="655.5" width="360" height="48"/>
+                                                                <constraints>
+                                                                    <constraint firstAttribute="height" relation="greaterThanOrEqual" constant="48" id="HTg-gw-BIr"/>
```

Comment:
I had to add height constraints to all the buttons in these views because we removed the default `contentEdgeInsets` that the old `baseStyles` had, which were basically giving these buttons their "height". Now they have a real height consistent with button heights in the new checkout. @dannyalright lmk if 48 points sounds good or we should use the default 44 points instead.

Reply:
I'm a fan of the 48pt!